build_tools/github_actions/runner/gcp/create_image.sh

```diff
@@ -24,7 +24,7 @@ INSTANCE_NAME="${INSTANCE_NAME:-github-runner-template-${RUNNER_TYPE}-${TIME_STR
 IMAGE_NAME="${INSTANCE_NAME/-template/}"
 ZONE="${ZONE:-us-central1-a}"
 PROJECT=iree-oss
-BASE_IMAGE="${BASE_IMAGE:-projects/ubuntu-os-cloud/global/images/ubuntu-2204-jammy-v20230114}"
+BASE_IMAGE="${BASE_IMAGE:-projects/ubuntu-os-cloud/global/images/ubuntu-2204-jammy-v20230531}"
```

Comment:
The image creation failed to update packages from the base image `v20230114`. Filed #13921 with the suspected reason.